src/docs/perf/rendering/shader.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Reduce shader compilation jank on mobile
+short-title: Shader jank
+description: What is shader jank and how to minimize it.
+---
+
+If the animations on your mobile app appear to be janky,
+but only on the first run, you can _warm up_ the
```

Comment:
It would be great to specify here or elsewhere what we mean by "first run". Is it the first time the user opens the app, and then the second open is fine? Is it every time the user opens the app from scratch. If I remember correctly, it's the former for Android and the latter for iOS.

You probably want to go into this detail somewhere else than in the first paragraph.

Reply:
@liyuqian, can you clarify?

Reply:
Filip's understanding is correct: Android is the former and iOS is the latter.